src/test/java/uk/gov/hmcts/reform/iahomeofficeintegrationapi/domain/entities/HomeOfficeInstructResponseTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.iahomeofficeintegrationapi.domain.entities;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("unchecked")
+public class HomeOfficeInstructResponseTest {
+    @Mock
+    MessageHeader messageHeader;
+    private HomeOfficeInstructResponse instructResponse;
+
+    @BeforeEach
+    void setUp() {
+        instructResponse = new HomeOfficeInstructResponse(
+            messageHeader, null
```

Comment:
Why set a null here?  Should try and avoid nulls in code everywhere.

Reply:
In regular 200 response, there will be no errorDetail object, where as only in 400, they are sending this object back in the response. I will add a scenario to check the object